<launch>
<!-- Nodelet Manager Camera Top -->
<node pkg="nodelet" type="nodelet" name="nodelet_manager_top" args="manager"/>

   <!-- UVCCam Nodelet Top -->
   <node pkg="nodelet" type="nodelet" name="camTop" args="load hiwr_camera_controller/HiwrCameraControllerNodelet nodelet_manager_top" output="screen">

     <rosparam file="$(find hiwr_camera_controller)/param/config_Laptop.yaml"/>

   </node>

   <!-- Hiwr tired node -->

   <node pkg="nodelet" type="nodelet" name="hiwr_hand_move" args="load hiwr_hand_move/HiwrHandMove nodelet_manager_top" output="screen">
   <param name="video_stream" type="str" value="/camTop/output_video" />
   </node>

    <node pkg="nodelet" type="nodelet" name="debug_display" args="load display_nodelet/Display_node nodelet_manager_top">
    <param name="video_stream" type="str" value="/camTop/output_video_test" />
    </node>

</launch>
